packages/py-moose-lib/requirements.txt

```diff
@@ -0,0 +1,25 @@
+certifi==2024.6.2
```

Comment:
We need all of this??

Reply:
Dependencies of the build dependencies `setuptools wheel twine`